surf-auth/surfAuthDebot.sol

```diff
@@ -0,0 +1,128 @@
+pragma ton-solidity >=0.35.0;
+pragma AbiHeader expire;
+pragma AbiHeader time;
+pragma AbiHeader pubkey;
+
+import "../Debot.sol";
+import "../Terminal.sol";
+import "../Menu.sol";
+import "../AddressInput.sol";
+import "../ConfirmInput.sol";
+import "../Upgradable.sol";
+import "../Sdk.sol";
+import "./Network.sol";
+import "./Base64.sol";
+import "./SigningBoxInput.sol";
+import "./UserInfo.sol";
```

Comment:
i think its better to use imports from git
import "https://raw.githubusercontent.com/tonlabs/DeBot-IS-consortium/main/Terminal/Terminal.sol";

Reply:
Extracted to include.sol